qpid-jms-client/src/main/java/org/apache/qpid/jms/message/JmsMessageIDBuilder.java

```diff
@@ -115,6 +119,20 @@ public String toString() {
         public static JmsMessageIDBuilder create(String value) {
             return valueOf(value.toUpperCase(Locale.ENGLISH)).createBuilder();
         }
+
+        /**
+         * Validates the value given maps to the built in message ID builders and
+         * return the builder enumeration that it maps to which can later be used
+         * to create builders of that type.
+         *
+         * @param value
+         * 		The name of one of the built in message ID builders.
+         *
+         * @return the enumeration value that maps to the built in builder.
+         */
+        public static BUILTIN validate(String value) {
+            return valueOf(value.toUpperCase(Locale.ENGLISH));
```

Comment:
The old create method could now call this.

Reply:
I fixed that and pushed a new update for this change